### PR TITLE
Set inflation and staked return to zero

### PR DESCRIPTION
**Pull request by doasync**
We do not get the expected zero value in the UI because the `minInflation` param is set to `0.025` by default and is used to calculate the inflation. We can just change the default `InflationParams` (and set minimum inflation to zero) instead of disabling calculation logic. The inflation calculation is a part of basic UI (polkadot-js/apps).

image

### CODE REVIEW

File: packages/apps-config/src/api/params/inflation.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const DEFAULT_PARAMS: InflationParams = {
   falloff: 0.05,
   idealStake: 0.5,
   maxInflation: 0.1,
-  minInflation: 0.025
+  minInflation: 0
```
Comment (AndreiNavoichyk) [inline packages/apps-config/src/api/params/inflation.ts:19]: As far as we set inflation to `0` in Node's code we need to be sure that it will be the same in the BlockViewer until we switch it back.

All these parameters are using to calculate Inflation on the fly. By setting `minInflation` to `0` you just allow it to be `0`. It is not enough for us. The easiest solution, for now, is to use 0% always.

Comment (doasync) [reply packages/apps-config/src/api/params/inflation.ts]: I've hardcoded an [empty](https://github.com/Cerebellum-Network/block-viewer/blob/5cdb938e5eed3150349bdd811dcf1b330a66bb73/packages/react-hooks/src/useInflation.ts#L16) value for the [inflation](https://github.com/Cerebellum-Network/block-viewer/blob/5cdb938e5eed3150349bdd811dcf1b330a66bb73/packages/page-staking/src/useSortedTargets.ts#L176) object: `{ inflation: 0, stakedReturn: 0 }`

File: packages/apps-config/src/api/chain/cere.ts [hunk 2]
```diff
@@ -6,20 +6,20 @@
 
 // For the network after ERC20 integrated
 export default {
-  ChainId: "u8",
-  DepositNonce: "u64",
-  ResourceId: "[u8; 32]",
+  ChainId: 'u8',
```
Comment (AndreiNavoichyk) [inline packages/apps-config/src/api/chain/cere.ts:9]: Do not include changes to PR that are not related to its main purpose

File: packages/apps-config/src/api/chain/index.ts [hunk 3]
```diff
@@ -1,13 +1,13 @@
 // Copyright 2017-2021 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import Phala from './phala';
 import Cere from './cere';
+import Phala from './phala';
```
Comment (AndreiNavoichyk) [inline packages/apps-config/src/api/chain/index.ts:5]: Do not include changes to PR that are not related to its main purpose

File: packages/page-settings/src/Developer.tsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ import { isJsonObject, stringToU8a, u8aToString } from '@polkadot/util';
 
 import { useTranslation } from './translate';
 
-const EMPTY_CODE = `{}`;
+const EMPTY_CODE = '{}';
```
Comment (AndreiNavoichyk) [inline packages/page-settings/src/Developer.tsx:18]: Do not include changes to PR that are not related to its main purpose

**Review by AndreiNavoichyk — APPROVED** [review]
